src/lib/shared_utils/src/canister_specific/individual_user_template/types/hot_or_not/mod.rs

```diff
@@ -826,14 +1038,16 @@ impl Post {
             VirtualMemory<DefaultMemoryImpl>,
         >,
     ) {
+        let slot_id_owned = slot_id_type.clone();
+
         let hot_or_not_details = self.hot_or_not_details.as_mut();
 
         if hot_or_not_details.is_none() {
             return;
         }
```

Comment:
is this relevant now or should we proceed to tabulate with inserting Some default ? In the context of old posts which were hotornotdetails - None